### PR TITLE
(maint) Update sync.yaml for new modsync config options

### PR DESCRIPTION
**Pull request by glennsarti**
Previously the travis.yaml file was unmanaged as it contained customisations not
compatible with modsync.  Now that the required options are available the sync
file can be updated accordingly.  This commit:
- Updates the sudo, dist and before_install commands to install Powershell
- Updates the NOTICE file to indicate the correct dates for the copyright
  holders

### CODE REVIEW

**Comment by glennsarti** [comment]
Travis.yaml is to remain unmanaged until https://github.com/puppetlabs/modulesync_configs/pull/178 is merged

**Comment by michaeltlombardi** [comment]
@glennsarti changes look fine, puppetlabs/modulesync_configs#178 is merged

File: .sync.yml [hunk 1]
```diff
@@ -8,7 +8,16 @@ Gemfile:
         platforms: ["mswin", "mingw", "x64_mingw"]
         version: '< 2.6.0'
 .travis.yml:
-  unmanaged: true
```
Comment (michaeltlombardi) [inline .sync.yml]: So this file's now unmanaged but we're adding settings for it?

Comment (glennsarti) [reply .sync.yml]: Correct.  We are now managing this file with the specified settings.

**Pull request merged**